### ISSUE TITLE
Passing is_collection option raises NoMethodError

### ISSUE THREAD

**Issue by chawes13** [LaunchPadLab/decanter#118]
## Environment

- `decanter` version: 3.4.1
- `ruby` version: 2.7.0
- `rails` version: 5.2.3

## Expected Behavior
- Passing in the `is_collection` option either treats the args as a collection or not, depending on the value

## Current Behavior
`NoMethodError` is thrown
```sh
pry(main)> Trip.decant([{ description: 'Hawaii' }], is_collection: true)
NoMethodError: undefined method `[]' for true:TrueClass
from /Users/chawes/Documents/LaunchPad/opex/decanter/lib/decanter/extensions.rb:57:in `is_collection?'
```

## Steps to Reproduce
- Call a decanter from an ActiveRecord model with the `is_collection` option provided (any value)

## Suggested Solution
- `is_collection?` should be modified to look at the value of `option[:is_collection]` instead of treating that value as a hash
  - Basically, the equivalent of `options.dig(:is_collection, :is_collection)` is happening
